### PR TITLE
deep-Blue theme

### LINKED ISSUE #115: New theme Deep-Blue
Created a new theme **deep Blue**

**Below is the snapshot of the theme**
image

**Color palette**

        Background  #165795FF
        Border      #BA9D6F
        Stroke      #38DD69FF
        Ring        #37DD57FF
        Fire        #1CD577FF
        SideNums    #120E5BF2
        SideLabels  #1ADD40FF
        Dates      #11E2E7FF
        CurrStreakNum  #639E29
        CurrStreakLabel #0FDD21FF
        

Hope you like it.
Thank you,

### PR DESCRIPTION
**Pull request by SuhasBRao**
## Description

This PR is regarding the new theme for which i have already created an issue. Please do review and let me know if you need any changes. I have already tested it locally.

Fixes #115  

### Type of change

- [ ] Bug fix (added a non-breaking change which fixes an issue)
- [x] New feature (added a non-breaking change which adds functionality)
- [ ] Updated documentation (updated the readme, templates, or other repo files)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?

- [x] Tested locally with a valid username
- [ ] Tested locally with an invalid username
- [ ] Ran tests with `composer test`
- [ ] Added or updated test cases to test new features

## Checklist:

- [x] I have checked to make sure no other [pull requests](https://github.com/DenverCoder1/github-readme-streak-stats/pulls?q=is%3Apr+sort%3Aupdated-desc+) are open for this issue
- [x] The code is properly formatted and is consistent with the existing code style
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

image

Thank you

### CODE REVIEW

File: docs/themes/README.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ You can also try out and customize these themes on the [Demo Site](https://githu
 |     `city-lights`      | ![image](https://user-images.githubusercontent.com/20955511/114738104-542b4b00-9d50-11eb-87cd-2446b4e5d1a7.png) |
 |         `blux`         | ![image](https://user-images.githubusercontent.com/73984717/116656177-7068f200-a9b6-11eb-91c5-5f0f1997720d.png) |
 |        `earth`         | ![image](https://user-images.githubusercontent.com/20955511/119410749-fcd6ae00-bcf1-11eb-8737-c78566e9e8f3.png) |
-
+|        `deepBlue`      | ![image](https://github-readme-streak-stats.herokuapp.com?user=SuhasBRao&stroke=38DD69&background=165795&ring=37DD57&fire=1CD577&sideNums=120E5BF2&sideLabels=1ADD40&currStreakLabel=10EF24&dates=11E2E7)
```
Comment (DenverCoder1) [inline docs/themes/README.md:70]:
```suggestion
|        `deepBlue`      | ![image](https://user-images.githubusercontent.com/20955511/127893911-23e6c0c6-2d97-4028-a2b9-1fa1063db776.png) |

```

**Review by DenverCoder1 — APPROVED** [review]
Thanks for contributing! :+1: 

Could you perhaps replace the URL with a static png link (as seen in the suggestion below), thanks!

**Comment by SuhasBRao** [comment]
Thanks for replying, Yes I will make the necessary changes.

**Comment by SuhasBRao** [comment]
I have added new link as required, please review once.

File: docs/themes/README.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ You can also try out and customize these themes on the [Demo Site](https://githu
 |     `city-lights`      | ![image](https://user-images.githubusercontent.com/20955511/114738104-542b4b00-9d50-11eb-87cd-2446b4e5d1a7.png) |
 |         `blux`         | ![image](https://user-images.githubusercontent.com/73984717/116656177-7068f200-a9b6-11eb-91c5-5f0f1997720d.png) |
 |        `earth`         | ![image](https://user-images.githubusercontent.com/20955511/119410749-fcd6ae00-bcf1-11eb-8737-c78566e9e8f3.png) |
-
+|        `deepBlue`      | ![image](https://user-images.githubusercontent.com/20955511/127893911-23e6c0c6-2d97-4028-a2b9-1fa1063db776.png)
```
Comment (DenverCoder1) [inline docs/themes/README.md:70]: Missing `|` and blank line after the table

```suggestion
|       `deepBlue`       | ![image](https://user-images.githubusercontent.com/20955511/127893911-23e6c0c6-2d97-4028-a2b9-1fa1063db776.png) |

```

**Review by DenverCoder1 — COMMENTED** [review]
Sorry to request changes again, but there's one last thing with the formatting

(You should be able to automatically commit it with the button under the suggestion)

**Comment by SuhasBRao** [comment]
> Sorry to request changes again, but there's one last thing with the formatting
> 
> (You should be able to automatically commit it with the button under the suggestion)

Thanks a lot for being patient and replying, I'm kind of new here. Sorry for the inconvenience.
Please let me know if there are any more changes to be done.

**Review by DenverCoder1 — APPROVED** [review]
Looks perfect! 👍

**Comment by DenverCoder1** [comment]
> Thanks a lot for being patient and replying, I'm kind of new here. Sorry for the inconvenience.

No problem at all

**Comment by DenverCoder1** [comment]
It looks like from the check, you are missing the color for `currStreakNum`

File: src/themes.php [hunk 3]
```diff
@@ -734,4 +734,15 @@
         "sideLabels" => "#C48519",
         "dates" => "#BA9D6F",
     ],
+    "deepBlue" => [
+        "background" => "#165795FF",
+        "border" => "#BA9D6F",
+        "stroke" => "#38DD69FF",
+        "ring" => "#37DD57FF",
+        "fire" => "#1CD577FF",
```
Comment (DenverCoder1) [inline src/themes.php:742]: Suggested fix based on previous commit (currStreakNum was removed)

```suggestion
        "fire" => "#1CD577FF",
        "currStreakNum" => "#639E29",
```

**Comment by SuhasBRao** [comment]
I have made the changes, Thanks a lot again.

**Pull request merged**